docs/rtd_requirements.txt

```diff
@@ -0,0 +1,14 @@
+boto3==1.7.5
```

Comment:
What's the strategy for keeping this up to date?

Reply:
Diligence? We could have a simple `make doc-requirements` that redirects the output of `pipenv lock --requirements --dev` to `docs/rtd_requirements.txt`.

Reply:
Either/or if it becomes a problem we can automate.